src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -4438,6 +4439,10 @@ namespace HeatBalanceManager {
 		SetupOutputVariable( "Zone Outdoor Air Drybulb Temperature [C]", Zone( ZoneLoop ).OutDryBulbTemp, "Zone", "Average", Zone( ZoneLoop ).Name );
 		SetupOutputVariable( "Zone Outdoor Air Wetbulb Temperature [C]", Zone( ZoneLoop ).OutWetBulbTemp, "Zone", "Average", Zone( ZoneLoop ).Name );
 		SetupOutputVariable( "Zone Outdoor Air Wind Speed [m/s]", Zone( ZoneLoop ).WindSpeed, "Zone", "Average", Zone( ZoneLoop ).Name );
+
+		// Hybrid model output added by Sang Hoon Lee August 2016
+		SetupOutputVariable("Zone Infiltration Hybrid Model Air Change Rate [ach]", Zone( ZoneLoop ).InfilOAAirChangeRateHM, "Zone", "Average", Zone(ZoneLoop).Name);
+
```

Comment:
I am not sure this is a good place to report Zone Infiltration Hybrid Model Air Change Rate at each zone time step. The infiltration results are reported at each system time step.

					SetupOutputVariable( "Zone Infiltration Air Change Rate [ach]", ZnAirRpt( Infiltration( Loop ).ZonePtr ).InfilAirChangeRate, "System", "Average", Zone( Infiltration( Loop ).ZonePtr ).Name );